hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHiveIncrementalPuller.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.hudi.utilities;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestHiveIncrementalPuller {
+
+  private HiveIncrementalPuller.Config config;
+
+  @Before
+  public void init() {
```

Comment:
rename to `setup()`? most of our tests are that way I think.

Reply:
Done.